labs-ios-starter/Profiles/View Controllers/ProfileDetailViewController.swift

```diff
@@ -101,6 +101,7 @@ class ProfileDetailViewController: UIViewController {
         firstNameLabel.text = profile.firstName
         lastNameLabel.text = profile.lastName
         emailLabel.text = profile.email
+        roundedImageView()
```

Comment:
Nice, these four little lines went a long way to improving the UI

Reply:
I also rearranged the views on the storyboard, but nothing drastic. Figured you could do more with it later if you want and we have time :smile: